java.md

```diff
@@ -0,0 +1,17 @@
+# Resources for JAVA
+
+## For Beginners
+
+1.  [Java For Beginner](https://www.youtube.com/watch?v=r59xYe3Vyks&list=PLS1QulWo1RIbfTjQvTdj8Y6yyq4R7g-Al)
+1.  [Learning References (GeeksForGeeks)](https://www.geeksforgeeks.org/java//)
+1.  [Code Monk](https://www.hackerearth.com/practice/)
```

Comment:
This is a generic programming resource, which does not belong here

Reply:
Ok bhaiya  I will work on some stuff then update